src/plugins/azgenerator/TemplateAzureCliParams.ts

```diff
@@ -69,6 +69,9 @@ export function GenerateAzureCliParams(model: CodeModelAz): string[] {
     if (hasLocation) parameterImports.push("get_location_type");
 
     header.addFromImport("azure.cli.core.commands.parameters", parameterImports);
+    if (hasLocation) {
+        header.addFromImport('azure.cli.core.commands.validators', ['get_default_location_from_resource_group']);
+    }
```

Comment:
should also check whether the command has the `resource_group_name` parameter.

Reply:
ohh right